modules/understanding-upgrade-channels.adoc

```diff
@@ -134,5 +134,21 @@ ifndef::openshift-origin[]
 [discrete]
 == Switching between channels
 
-Your cluster is still supported if you change from the `stable-{product-version}` channel to the `fast-{product-version}` channel. Although you can switch to the `candidate-{product-version}` channel at any time, some releases in that channel might be unsupported release candidates. You can switch from the `candidate-{product-version}` channel to the `fast-{product-version}` channel if your current release is a general availability release. You can always switch from the `fast-{product-version}` channel to the `stable-{product-version}` channel, although if the current release was recently promoted to `fast-{product-version}` there can be a delay of up to a day for the release to be promoted to `stable-{product-version}`. If you change to a channel that does not include your current release, an alert displays and no updates can be recommended, but you can safely change back to your original channel at any point.
+A channel can be switched from the web console through the `adm upgrade channel` command:
+
+----
+$ oc adm upgrade channel clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>”}]'
+----
+
+If you change to a channel that does not include your current release, an alert will display and no updates can be recommended, but you can safely change back to your original channel at any point.
+
+Changing your channel might impact the supportability of your cluster. The following conditions might apply:
+
+* Your cluster is still supported if you change from the `stable-{product-version}` channel to the `fast-{product-version}` channel.
```

Comment:
This line, appears as (in the preview)
Your cluster is still supported if you change from the _stable-Branch Build_ channel to the _fast-Branch Build_ channel.
Will "_stable-Branch Build_ channe" & the "_fast-Branch Build_" be something like "stable-4.8" & "fast-4.8" etc?

Reply:
Yes it will. I am new (starting my 3rd month), so I don't have the ability to do that yet. Once it is merged "_stable-Branch Build_ channel" and "_fast-Branch Build_" will reflect the version of the docs the changes are made in.